### PR TITLE
Add the resistor-color-duo exercise.

### PR DESCRIPTION
**Pull request by MichaelBunker**
See the specification: https://github.com/exercism/problem-specifications/tree/main/exercises/resistor-color-duo

### CODE REVIEW

File: exercises/practice/resistor-color-duo/.meta/example.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * <link>.
+ *
+ * To disable strict typing, comment out the directive below.
+ */
+
+declare(strict_types=1);
+
+class ResistorColorDuo
+{
+    public const COLORS = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"];
```
Comment (neenjaw) [inline exercises/practice/resistor-color-duo/.meta/example.php:29]: I think this is the right implementation as it is built from the last exercise, could you look at adding some hints to the previous exercise for students to use a constant here? Or maybe we shouldn't use a public array constant at all, and specify another method to list the colors to make the implementation and learning goals more obvious?

**Review by neenjaw — APPROVED** [review]
Looks good, once you look at the feedback for https://github.com/exercism/php/pull/534 I think a few of the same comments can be applied here, but I don't want to nag.

Happy to merge this optimistically, let me know what you'd like to do

File: exercises/practice/resistor-color-duo/.meta/example.php (hunk 1 above)
Comment (MichaelBunker) [reply exercises/practice/resistor-color-duo/.meta/example.php]: I changed it to a method to make it more obvious. seeing just `COLORS` in the test file might be confusing for someone just starting out.

**Comment by MichaelBunker** [comment]
@neenjaw I've made changes that you requested to resistor-color and resistor-color-duo. This is ready for review.

**Review by neenjaw — APPROVED** [review]

**Pull request merged**